cli/cmd/cluster.go

```diff
@@ -47,6 +47,7 @@ var (
 	_flagClusterEnv       string
 	_flagClusterConfig    string
 	_flagClusterInfoDebug bool
+	_flagClusterYes       bool
```

Comment:
nit: rename to _flagClusterSkipPrompt? In the middle of code, it's not obvious what this variable represents